### PR TITLE
Add QAI service

### LINKED ISSUE #4: QAI container not implemented

### PR DESCRIPTION
**Pull request by Brutus5000**
fixes #4, but depends on https://github.com/FAForever/QAI/pull/56

### CODE REVIEW

**Comment by Brutus5000** [comment]
I have tested it on a local vm with faf-stack. The service fails because API keys etc. are wrong, but it starts.

**Comment by Brutus5000** [comment]
@micheljung Can we merge this?

File: docker-compose.yml [hunk 1]
```diff
@@ -233,6 +233,20 @@ services:
       - "8167:8167"
 
   #
+  # QAI irc bot
+  #
+  faf-qai:
+    container_name: faf-qai
+    build: git://github.com/FAForever/QAI
```
Comment (micheljung) [inline docker-compose.yml]: Please reference a specific commit or tag, otherwise we never know what we actually get and it might or might not work.

**Comment by Brutus5000** [comment]
Right you are and fixed.

**Pull request merged**